src/main/resources/assets/ingameinfo/lang/de_DE.lang

```diff
@@ -1,45 +1,284 @@
+# https://github.com/ReneMuetti/InGame-Info-XML
+
 # gui - config - categories
+ingameinfoxml.config.category.general=Allgemein
+ingameinfoxml.config.category.general.tooltip=Allgemeine Optionen.
+ingameinfoxml.config.category.alignment=Ausrichtung
+ingameinfoxml.config.category.alignment.tooltip=Ausrichtungs-Versatz.
 
 # gui - config - general
+ingameinfoxml.config.filename=Dateiname
+ingameinfoxml.config.filename.tooltip=Die Konfiguration, welche beim Systemstart geladen werden soll.
+ingameinfoxml.config.replaceDebug=Ersetze Debug Overlay
+ingameinfoxml.config.replaceDebug.tooltip=Ersetzt den Debug-Overlay (F3) mit dem InGameInfoXML Overlay.
+ingameinfoxml.config.showInChat=Zeige im Chat
+ingameinfoxml.config.showInChat.tooltip=Zeige den Overlay im Chat.
+ingameinfoxml.config.showOnPlayerList=Auf der Spieler Liste anzeigen
+ingameinfoxml.config.showOnPlayerList.tooltip=Zeige den Overlay auf der Spieler Liste.
+ingameinfoxml.config.scale=Skalierung
+ingameinfoxml.config.scale.tooltip=Das Overlay wird um diesen Betrag skaliert.
+ingameinfoxml.config.fileInterval=Datei-Tag-Intervall
+ingameinfoxml.config.fileInterval.tooltip=Der Intervall zwischen beim Datei lesen für den 'file' Tag (in Sekunden).
 
 # gui - config - alignment
+ingameinfoxml.config.topleft=Oben Links
+ingameinfoxml.config.topleft.tooltip=Offsets für die obere linke Ecke des Bildschirms (X <space> Y).
```

Comment:
`X<space>Y` is supposed to mean `X Y` so it should be `X<german translation for 'space'>Y`